drivers/gles2/rasterizer_storage_gles2.cpp

```diff
@@ -3236,12 +3237,14 @@ Color RasterizerStorageGLES2::multimesh_instance_get_custom_data(RID p_multimesh
 void RasterizerStorageGLES2::multimesh_set_as_bulk_array(RID p_multimesh, const PoolVector<float> &p_array) {
 	MultiMesh *multimesh = multimesh_owner.getornull(p_multimesh);
 	ERR_FAIL_COND(!multimesh);
+	ERR_FAIL_COND(!multimesh->data.ptrw());
```

Comment:
Didn't you mean checking `!multimesh->data` here?

Reply:
No, data variable is Vector<float> type, and when Vector is empty ptrw() returns NULL.
I could use condition - `ERR_FAIL_COND(multimesh->data.size() == 0);` but I want to do it in a similar way as it is used - copymem function in line 3248.

Reply:
`ptrw()` does a copy on write though, the "w" stands for "write" as it's the one you should use to write data. If you only want to check that there is data, I think you can use `ptr()` (or indeed `.size() == 0`).

Reply:
Done